src/components/StepIndicator/StepIndicatorStep.styles.ts

```diff
@@ -4,9 +4,9 @@ export const baseCss: ThemeCss = theme => ({
   display: `inline-flex`,
   alignItems: `center`,
   borderRadius: `${theme.radii[2]} ${theme.radii[2]} 0 0`,
-  background: theme.colors.secondaryBackground,
+  backgroundColor: theme.colors.grey[5],
   border: `1px solid ${theme.colors.standardLine}`,
-  color: theme.colors.grey[40],
+  color: theme.colors.grey[60],
```

Comment:
```suggestion
  color: theme.colors.grey[50],
```

would give us a color contrast of `4.39`, which seems fine for this text size (`18px`), complies to AA+ / passes the Storybook a11y test?

Reply:
Ah no, it's not — I'm blind. ;-) Mhh, let's see if we can get a this a bit more deemphasized via `blackFade` then … :P